tests/unit/react/single.js

```diff
@@ -94,9 +94,13 @@ TestUtil.createReactTest(
     ['inputs/single', 'templates'],
     { defaultValue: 'Amsterdam', items: ['Amsterdam', 'Antwerp', 'Athens'] },
     function(SelectivityReact, test, ref) {
+        var el = ref._reactInternalInstance._hostContainerInfo._node;
+
         test.deepEqual(ref.getData(), { id: 'Amsterdam', text: 'Amsterdam' });
 
         test.deepEqual(ref.getValue(), 'Amsterdam');
+
+        test.equal(el.querySelector('input').value, 'Amsterdam');
```

Comment:
You can get two more parameters in your test function: `container` and `$`. `container` is the `el` you've retrieved from the reference (or its parent, not sure by head :) and `$` is actually a shortcut for `container.querySelectorAll`. I think it'd be nicer if you use those instead of relying on React internals.